pipeval/validate/__main__.py

```diff
@@ -15,24 +14,19 @@ def positive_integer(arg):
 
     return i
 
-def _parse_args():
+def add_subparser_validate(subparsers:argparse._SubParsersAction):
     """ Parse arguments """
-    parser = argparse.ArgumentParser()
-    parser.add_argument('path', help='one or more paths of files to validate', type=str, nargs='+')
-    parser.add_argument('-v', '--version', action='version', version=f'%(prog)s {__version__}')
+    parser = subparsers.add_parser(
+        name = 'validate',
+        help = 'Validate one or more file(s)',
+        description = 'Validate one or more file(s)',
+        formatter_class = argparse.ArgumentDefaultsHelpFormatter
+    )
+
+    parser.add_argument('path', help='One or more paths of files to validate', type=str, nargs='+')
```

Comment:
It's not a critical point, but for input paths I generally use something like [this](https://stackoverflow.com/a/11541495) `extant_file` type rather than `str`.

```python
def extant_file(x):
    """
    'Type' for argparse - checks that file exists but does not open.
    """
    if not os.path.exists(x):
        # Argparse uses the ArgumentTypeError to give a rejection message like:
        # error: argument input: x does not exist
        raise argparse.ArgumentTypeError("{0} does not exist".format(x))
    return x
```

Reply:
That's a good point; since the tool is intended to validate files, the existence check so far has been put into the actual validation rather than letting it happen through argparse. I'll add an issue for moving the existence validation to happen through the argument though!